src/main/java/io/mosip/print/service/impl/PrintRestClientServiceImpl.java

```diff
@@ -83,30 +82,25 @@ public Object getApi(ApiName apiName, List<String> pathsegments, String queryPar
 			try {
 
 				uriComponents = builder.build(false).encode();
-				printLogger.debug(uriComponents.toUri().toString(), "URI", "", "");
+				printLogger.debug(uriComponents.toUri().toString()+ "URI");
 				obj = restApiClient.getApi(uriComponents.toUri(), responseType);
 
 			} catch (Exception e) {
 				e.printStackTrace();
-				printLogger.error(LoggerFileConstant.SESSIONID.toString(),
-						LoggerFileConstant.REGISTRATIONID.toString(), "",
-						e.getMessage() + ExceptionUtils.getStackTrace(e));
-
+				printLogger.error(e.getMessage() + ExceptionUtils.getStackTrace(e));
```

Comment:
Do not use concatenation. To print stackstrace use as mentioned here. 
`printLogger.error(e.getMessage(), e);`

Reply:
taken care